main.go

```diff
@@ -330,10 +333,14 @@ func genJSStubs(gen *protogen.Plugin, file *protogen.File,
 
 			// If the input comes from an outside package, we need
 			// to prepend the outside package's name to the type.
-			// This will likely be a "manual_import" that the user
-			// has specified.
-			// TODO: remove the need for manual import?
 			if inputPkg != pkg {
+				// If this import is the same as the manual
+				// import, then there is no need to add it.
+				if typeImportPath != manualImport {
```

Comment:
Could we instead just put the manual import into the `typeImportPath`? And since it's a map, it would just be overwritten if it's already there because of a normal import? Then we could remove that part from the template and just use the `.AdditionalImports`.

Reply:
good idea!